README.md

```diff
@@ -57,4 +67,32 @@ curl -v -X POST -H "Content-Type: application/json" http://localhost:9050/event-
       filterPath = "$.event[?(@.enrolment =~ /HMRC\\-CUS\\-ORG\\~EORINumber~.*/i)]"
     }
   }
-}```
+}
+```
+
+### Example for adding configuration for topics in `app-config-$env` files
+#### below example shows configuration for topic `email` that has 2 subscribers `channel-preferences` and `customs-data-store`
+
+```
+    topics.email.channel-preferences.uri: https://channel-preferences.protected.mdtp:443/channel-preferences/process/bounce
+    topics.email.channel-preferences.http-method: POST
+    topics.email.channel-preferences.elements: 100
+    topics.email.channel-preferences.per: 3.seconds
+    topics.email.channel-preferences.max-connections: 4
+    topics.email.channel-preferences.min-back-off: 5.millis
+    topics.email.channel-preferences.mix-back-off: 10.millis
+    topics.email.channel-preferences.max-retries: 10.millis
+    topics.email.channel-preferences.filterPath: $.event[?(@.enrolment =~ /HMRC\\-CUS\\-ORG\\~EORINumber~.*/i)]
+
+    topics.email.customs-data-store.uri: https://channel-preferences.protected.mdtp:443/channel-preferences/process/bounce
```

Comment:
did you mean to re-use the same uri a channel-preferences for the example?

Reply:
no